exercise_3/Course.hpp

```diff
@@ -0,0 +1,28 @@
+//
+// Created by denys on 10.07.19.
+//
+
+#pragma once
+
+
+#include "ARecord.hpp"
+
+class Course : public ARecord {
+private:
+    int _id;
+    int _teacher_id;
+    std::string _name;
+
+    std::list<std::pair<std::string, std::string>> _get_print_parameters() const override;
+
+public:
+    static const std::string MARKER;
```

Comment:
Why public?
Brakes encapsulation (from the current usage perspective, at least).

Reply:
I use this MARKER outside the Reccords. ( RecordsHandler.cpp:36).

Reply:
Yes, got it.
However, that indicates a teeny tiny code smell.  The usage of those markers in RecordsHandler isn't bounded to the actual state of an object... would consider that particular piece of code whether implementation change isn't beneficial.